engine/reference/commandline/build.md

```diff
@@ -94,11 +96,9 @@ Build Syntax Suffix             | Commit Used           | Build Context Used
 
 If you pass an URL to a remote tarball, the URL itself is sent to the daemon:
 
-Instead of specifying a context, you can pass a single Dockerfile in the `URL`
-or pipe the file in via `STDIN`. To pipe a Dockerfile from `STDIN`:
-
```

Comment:
This was done in 7b41a45abe004f7d5a7d69f1eba6650f28e68e76 which may need to be reverted in `master`. I'll check into it.